tests/functional/api/BackbeatServer.js

```diff
@@ -573,12 +573,6 @@ describe('Backbeat Server', () => {
                 VersionId: 'c',
                 StorageClass: 'd',
             }],
-            [{
```

Comment:
I think it would make sense to keep this check when on non-NFS bucket, and possibly add a check on NFS bucket expecting success without versionId.

Reply:
I agree, although the API cannot be aware of whether it is an retrying an NFS bucket as it just pushes the information the the kafka topic and responds that CRR is PENDING.